### ISSUE TITLE
Upgrade hanami to the latest version

### ISSUE THREAD

**Issue by brunoarueira** [feed-the-hungry/backend#123]
The last published version is 2.0.3 and this project should be upgraded to that.

**Comment by brunoarueira** [comment]
Since hanami-modelo isn't compatible, this issue will be on hold.

**Comment by brunoarueira** [comment]
Blog post to help upgrade: https://loyalty.dev/posts/upgrading-hanami-part-one

Thread from official hanami discourse: https://discourse.hanamirb.org/t/hanami-1-3-hanami-2-0/702/2

**Comment by brunoarueira** [comment]
After #140, probably the next step is to work on this issue

**Comment by brunoarueira** [comment]
Fixed at #146
